parachains/integration-tests/collectives/0_xcm/0_init.yml

```diff
@@ -62,13 +62,26 @@ tests:
               pallet: xcmPallet
               call: send
               args: [
-                { v1: { 0, interior: { x1: { parachain: *cp_id }}}}, # destination
+                { v3: { 0, interior: { x1: { parachain: *cp_id }}}}, # destination
                 {
-                  v2: [ # message
+                  v3: [ # message
+                    {
+                      UnpaidExecution: {
+                        weightLimit: { 
+                          limited: {
+                            refTime: 2200000000, # 2_200_000_000
+                            proofSize: 200000, # 200_000
+                          },
+                        }
+                      }
+                    },
                     {
                       Transact: {
-                        originType: Superuser,
-                        requireWeightAtMost: 1000000000, # 1_000_000_000
+                        originKind: Superuser,
+                        requireWeightAtMost: {
+                          refTime: 200000000, # 200_000_000 
```

Comment:
Curious where this values are coming from, seems different to what it used to be, but also the weight format has changed.